hypershift-operator/controllers/hostedcluster/metrics/metrics.go

```diff
@@ -104,7 +107,7 @@ var (
 		configv1.IdentityProviderTypeRequestHeader,
 	}
 
-	knownPlatforms = hyperv1.PlatformTypes()
+	knownPlatforms = hyperv1beta1.PlatformTypes()
```

Comment:
fwiw we use hyperv1 across the code base, that alias would save a lot of replacements when there's an api bump.

Reply:
fixed